swagger.yml

```diff
@@ -167,13 +159,13 @@ paths:
         "201":
           description: "Item successfully added"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+           description:  "Authentication information is missing or invalid"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+           description:  "User is not authorised to make the request"
         "405":
           description: "Method not allowed"
         "503":
-          $ref: '#/responses/InternalServerError'
+           description:  "User is not authorised to make the request"
```

Comment:
Please use the parameterised values here, there is no point repeating the messages again. The last block has the common 401, 403 and 503 parameterised